doc/cascadia/profiles.schema.json

```diff
@@ -457,109 +458,31 @@
-                "description": "Sets the color used as ANSI bright black."
-              },
-              "brightBlue": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI bright blue."
-              },
-              "brightCyan": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI bright cyan."
-              },
-              "brightGreen": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI bright green."
-              },
-              "brightPurple": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI bright purple."
-              },
-              "brightRed": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI bright red."
-              },
-              "brightWhite": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI bright white."
-              },
-              "brightYellow": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI bright yellow."
-              },
-              "cyan": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI cyan."
-              },
-              "foreground": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the foreground color of the color table."
-              },
-              "green": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI green."
-              },
-              "purple": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI purple."
-              },
-              "red": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI red."
-              },
-              "white": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI white."
-              },
-              "yellow": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI yellow."
-              }
-            },
-            "type": "object"
-          },
-          "type": "array"
-        },
         "commandline": {
           "description": "Executable used in the profile.",
           "type": "string"
         },
         "connectionType": {
           "$ref": "#/definitions/ProfileGuid",
-          "description": "A GUID reference to a connection type. Currently undocumented as of 0.3, this is used for Azure Cloud Shell"
+          "description": "A GUID reference to a connection type."
```

Comment:
```suggestion
          "description": "A GUID reference to a connection type. The list of available connection types is a jealously-guarded secret."
```

Reply:
Don't say exactly that, but _I AM NOT DOCUMENTING THEM THEY ARE NOT AN API_

Reply:
Could we just remove it from the schema/docs and pretend it doesn't exist?

Reply:
If it's not in the schema/docs it might show up as an _error_.

Reply:
I mean though, no one _should_ be setting this, right? You should only be using this on the auto-generated ACS profile, and that'll hind the actual value from the user, right?

Reply:
```suggestion
          "description": "A GUID reference to a connection type. This is only used for the Azure Cloud Shell."
```

Reply:
The part that's confusing to me is that an auto-generated ACS profile looks like this:
```js
{
            "name": "Azure Cloud Shell",
            "guid": "{b453ae62-4e3d-5e58-b989-0a998ec441b8}",
            "source": "Windows.Terminal.Azure"
}
```
so the user never sees a `connectionType`.

defaults-universal.json shows a `connectionType` for the telnet profile, but the schema isn't imported there.

So, if nobody should be setting this, why even include it in the schema at all?

Reply:
Good question. Remove it.

Just so long as that doesn't make connections that _do_ have connectionType look invalid.

It's 100% possible to create a telnet connection using only profiles.json.